doc/languages-frameworks/python.section.md

```diff
@@ -1272,6 +1272,10 @@ where `buildPythonPath` determines `program_PATH` and `program_PYTHONPATH`. The
 This also helps checking whether the package assumes dependencies such as
 `setuptools` at runtime, which is commonly not listed.
 
+If your package does not propagate all needed dependencies _(e.g. it has
+`passthru.optional-dependencies`)_ consider using the relaxed check
```

Comment:
How is this meant to function? For imports to succeed those optional dependencies will have to be added to the build inputs. The passthru does not go to the derivation so it can't be aware of it. Optional dependencies *should not* be in `*buildInputs`.